synapse/common.py

```diff
@@ -973,6 +973,12 @@ def unjsonsafe_nodeedits(nodeedits):
 
     return retn
 
+def reprauthrule(rule):
```

Comment:
can this be used by `synapse.lib.stormlib.auth.LibAuth.textFromRule`  ?

Reply:
Yeah that's where I grabbed it from and forgot to swap that one out too, will fix